ProQuest.js

```diff
@@ -118,59 +121,95 @@ function fetchEmbeddedPdf(url, item, callback) {
 		}
 	}, callback);
 }
+*/
 
 function getSearchResults(doc, checkOnly) {
-	var tabs = doc.getElementsByClassName('tabContent');
 	var root;
-	if (tabs.length) {
-		for (var i = 0; i < tabs.length; i++) {
-			if (tabs[i].offsetHeight) {
-				if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
-				root = tabs[i].getElementsByClassName('resultListContainer')[0];
+	//AURIMASV will change the logic here accordingly to his github issue comments
+	// for old, deprecated search results format with tabs
+	// and we can probably remove it later
+	var tabs = doc.getElementsByClassName('tabContent');
+	for (var i = 0; i < tabs.length; i++) {
+		if (tabs[i].offsetHeight) {
+			if (Zotero.isBookmarklet && tabs[i].id != 'allResults-content') return false;
+			root = tabs[i].getElementsByClassName('resultListContainer')[0];
+			break;
+		}
+	}
+	if (!root) {
+		var elements = doc.getElementsByClassName('resultListContainer');
+		for (var i=0; i<elements.length; i++) {
+			if (elements[i] && elements[i].offsetHeight>0) {
+				root = elements[i];
 				break;
 			}
 		}
-	} else {
-		root = doc.getElementsByClassName('resultListContainer')[0];
-		if (root && !root.offsetHeight) return false; // Not sure if this can actually happen
 	}
-
-	if (!root) return false;
+	if (!root) {
+		Z.debug("No root found");
+		return false;
+	}
 
 	var results = doc.getElementsByClassName('resultItem');
```

Comment:
change this to `var results = root.querySelectorAll('.resultTitle, .previewTitle')`